addon/validators/format.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * For code taken from ember-cp-validations
+ * Copyright 2016, Yahoo! Inc.
+ * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
+ */
+
+import Ember from 'ember';
+import buildMessage from 'ember-changeset-validations/utils/validation-errors';
+
+const {
+  isEmpty
```

Comment:
This can be a oneliner since it's only destructuring one thing. Generally prefer to only split when > 3 destructures